actions/push-to-gar-docker/action.yaml

```diff
@@ -77,6 +78,14 @@ inputs:
       Delete the credentials file after the action is finished.
       If you want to keep the credentials file for a later step, set this to false.
     default: "true"
+  secrets:
+    description: |
+      List of secrets to expose to the build.
+      Building from the current repository automatically uses the GitHub token, so it does not need to be passed.
+      If you want to authentiated against a different private repository, you will need to pass a secret named `GIT_AUTH_TOKEN` to be able to authenticate against it with Buildx.
+      `secrets: |
```

Comment:
Same as the example included in the description, all other inputs don't include examples, we should follow the same pattern here.

Since inputs are mapped one-to-one we can just reference the https://github.com/docker/build-push-action documentation.